### DISCUSSION TITLE
Packagist maintainership

### DISCUSSION THREAD

**Discussion by spawnia** [webonyx/graphql-php#1057] [General]
Hi @vladar, can you add me a as a maintainer on https://packagist.org/packages/webonyx/graphql-php? I recently messed up the release of `v14.11.5` by accidentally tagging `master` and would like to head in there and fix it 🙈

**Comment by vladar** [comment]
Hi, @spawnia Thanks so much for keeping the lib up-to-date. I desperately needed some break but plan to put some effort back into the lib in the future. In the meantime added you to the packagist maintainers 👍

**Reply by spawnia** [reply]
Thank you. I was able to update the broken release.

I am happy to have the chance to work on this library, it has been great. We are closing in on maximum correctness with strict static analysis, restoring test and feature parity with graphql-js, and have made some nice performance improvements. My plan is to cut a v15 somewhat soon.

**Reply by ruudk** [reply]
Thank you @spawnia for your amazing improvements 💙

**Reply by shmax** [reply]
Absolutely. I'm in awe of your diligence and dedication. Will buy you a beer or maybe a puppy when you max-out the phpstan level and kill off the baseline file once and for all.

**Reply by shmax** [reply]
By the way, I'm still available to help with stuff if you can think of something I can handle.
